conf/igenomes.config

```diff
@@ -0,0 +1,191 @@
+/*
+ * -------------------------------------------------
```

Comment:
If we're never going to use iGenomes I think we should ditch this file and the params that refer to igenomes stuff..